docs/mesos/ManualInstallCalicoMesos.md

```diff
@@ -0,0 +1,80 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.18.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Manually Install Calico and Mesos on Agent
+This guide explains how to add Calico networking to a Mesos slave.
+Specifically, we will add the Calico plugin to perform networking
+for the unified containerizer.
+- For information on the Docker Containerizer, see
+[Docker Containerizer Guide](UsageGuideDockerContainerizer.md).
+
+
+## Prerequisites
+### Mesos-Slave with Mesos and Netmodules installed
+This guide will focus on only the Calico components of a
+calico-mesos-slave. You must have a slave with net-modules
+activated. See the [Manual Net-Modules install Guide](net-modules/README.md).
```

Comment:
Can also link them to Mesos Slave setup (https://open.mesosphere.com/getting-started/install/#slave-setup). Probably the only time we should refer to it as a slave too...